doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -144,6 +144,16 @@ KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
 Bluetooth LE
 ============
 
+.. rst-class:: v2-7-0 v2-6-2 v2-6-1 v2-6-0 v2-5-3 v2-5-2 v2-4-4 v2-4-3
+
+DRGN-23231: The Bluetooth subsystem may sometimes deadlock when a Bluetooth link is lost during data transfer.
+  When this happens, the disconnected event is never delivered to the application.
+  The issue occurs when the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is enabled.
+  This option is enabled by default on the nRF5340 DK.
+  The issue is a result of the fix applied in DRGN-21085.
```

Comment:
```suggestion
DRGN-23231: The Bluetooth subsystem may sometimes deadlock when a Bluetooth link is lost during data transfer
  When this happens, the disconnected event is never delivered to the application.
  The issue occurs when the :kconfig:option:`CONFIG_BT_HCI_ACL_FLOW_CONTROL` Kconfig option is enabled.
  This option is enabled by default on the nRF5340 DK.
```
We do not use full stop on the first line (the short description of the issue.
And I think we do not want to advertise that an issue occurs due to a fix done earlier.